azurelinuxagent/common/protocol/wire.py

```diff
@@ -854,10 +854,25 @@ def update_goal_state(self, force_update=False):
 
     def _fetch_vm_settings_goal_state(self, force_update):
         """
-        Queries the vmSettings from the HostGAPlugin and returns an instance of ExtensionsGoalStateFromVmSettings.
-        Raises _VmSettingsNotSupportedError if the HostGAPlugin does not support FastTrack or ProtocolError on other kinds of errors.
+        Queries the vmSettings from the HostGAPlugin and returns an (ExtensionsGoalStateFromVmSettings, bool) tuple with the vmSettings and
+        a boolean indicating if they are an updated (True) or a cached value (False).
+
+        Raises ProtocolError if the request fails for any reason (e.g. not supported, time out, server error)
         """
-        etag = None if force_update or self._vm_settings_goal_state is None else self._vm_settings_goal_state.etag
+        def raise_not_supported(reset_state=False):
+            if reset_state:
+                self._host_plugin_supports_vm_settings = False
+                self._host_plugin_supports_vm_settings_next_check = datetime.now() + timedelta(hours=6)  # check again in 6 hours
+                # "Not supported" is not considered an error, so don't use self._vm_settings_error_reporter to report it
+                logger.info("vmSettings is not supported")
+                add_event(op=WALAEventOperation.HostPlugin, message="vmSettings is not supported", is_success=True)
```

Comment:
Shouldn't the op be VMSettings?

```suggestion
                add_event(op=WALAEventOperation.VmSettings, message="vmSettings is not supported", is_success=False)
```

Reply:
I used VmSettings for all the telemetry we need to monitor (errors, number of requests) and HostGAPlugin for informational purposes (HostGAPlugin version, vmSettings supported)